posts/2019-09-18-upcoming-docsrs-changes.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Upcoming docs.rs changes"
+author: The Rust Infrastructure Team
+---
+
+On September 30th breaking changes will be deployed to the [docs.rs] build
```

Comment:
I'd rephrase this to emphasize the benefits first.

---

On September 30th, we'll be upgrading docs.rs to use a stronger sandbox when building documentation. This change is not expected to effect the vast majority of users, but it is technically a breaking change, so we want to be sure everyone is aware of it.

## what is docs.rs?

[docs.rs] is a free service building and hosting documentation ... (as before)

## What will change

...